tests/report/display/main.fmf

```diff
@@ -0,0 +1,2 @@
+summary: Check for display output
+tag: []
```

Comment:
Why are all inherited tags removed?

Reply:
Copied from `/tests/report/html`, therefore "no idea" is the answer here. Dropped in 8d37bce3.